cql3/Cql.g

```diff
@@ -518,6 +518,15 @@ usingClauseObjective[std::unique_ptr<cql3::attributes::raw>& attrs]
     | K_TIMEOUT to=term { attrs->timeout = to; }
     ;
 
+usingClauseNoTTL[std::unique_ptr<cql3::attributes::raw>& attrs]
+    : K_USING usingClauseNoTTLObjective[attrs] ( K_AND usingClauseNoTTLObjective[attrs] )*
+    ;
+
+usingClauseNoTTLObjective[std::unique_ptr<cql3::attributes::raw>& attrs]
+    : K_TIMESTAMP ts=intValue { attrs->timestamp = ts; }
+    | K_TIMEOUT to=term { attrs->timeout = to; }
```

Comment:
I wonder (but maybe it's not important to think about this right now) whether it's better to call this "usingClauseNoTTL" or "UsingClauseTimestampOrTimeout". If we later add a new "USING XYZ", are we more likely to want to allow it in this one or not?

Reply:
I'm not sure at this point.
Like I said above, the only attributes applicable to writes are TIMESTAMP and TTL.
TTL itself is specific to INSERT/UPDATE that may set the TTL on the upserted entity
and `usingClauseNoTTL` makes sense for delete-like queries such as DELETE and TRUNCATE,
for which both the TIMESTAMP and TIMEOUT are applicable.
Maybe calling it as you suggested is the right thing to do at the moment.

Reply:
Maybe we should have a Santa Clause